backend/ethereum/glue/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2021 - See NOTICE file for copyright holders.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package glue glues together go-perun and go-ethereum.
+// It contains generic functionality and can he used even without go-perun.
+package glue // import "perun.network/go-perun/backend/ethereum/glue"
```

Comment:
The package name "glue" is very vague. Use package name "subscription" instead?

Reply:
Renamed it to `subscription` for now. I also wanted to put the reorg-resistant TX stuff in here, then the name `subscription` would not be enough.